activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   Fix `transaction` reverting for migrations.
+
+    Before: Commands run inside a `transaction` in a reverted migration ran uninverted.
+    Now: This change fixes that by reverting commands inside `transaction` block.
+
+    *fatkodima*, *dv*
```

Comment:
Looking at the original PR dv goes by "David Verhasselt" in the CHANGELOG so can you update that for consistecy?

Also can you update your commit to add the following so that he gets the commit credit in the contributors app too?

```
commit message text

[fatkodima & David Verhasselt]
```

Reply:
done.
Now this PR is failing (as opposed to before) due to various `apt-get install` failures ;(
I noticed similar failures for a major part of recent PRs. Is there any way to trigger travis run without dummy code changes?